redaxo/src/addons/media_manager/lib/effects/effect_convert2img.php

```diff
@@ -130,8 +178,14 @@ public function getParams()
     {
         $imNotfound = '';
         if (!class_exists(Imagick::class) && '' == self::getConvertPath()) {
-            $imNotfound = '<strong>'.rex_i18n::msg('media_manager_effect_convert2img_noimagemagick').'</strong>';
+            $imNotfound = '<strong>' . rex_i18n::msg('media_manager_effect_convert2img_noimagemagick') . '</strong> ';
         }
+
+        $videoConverterNotfound = '';
+        if (!$this->isFfmpegAvailable()) {
+            $videoConverterNotfound = '<strong>' . rex_i18n::msg('media_manager_effect_convert2img_videoconverternotfound') . '</strong> ';
```

Comment:
Die Variable scheint nicht weiter verwendet zu werden. Die Meldung erscheint also nirgends.

Reply:
die ausgäbe wird nun in einem array gespeichert